src/odemis/gui/win/acquisition.py

```diff
@@ -766,6 +766,14 @@ def __init__(self, parent, orig_tab_data):
         for sc in self.streambar_controller.stream_controllers:
             self._orig_entries += get_local_settings_entries(sc)
 
+        # range for the autofocus (in m)
+        # default min/max range to 50 µm (half a grid square) to 1 mm (half a grid)
+        # this range should cover all cases for different magnifications (e.g. 20x, 50x, 100x)
+        self.focus_points_dist_ctrl.SetValueRange(50e-6, 1000e-6)
+        self.focus_points_dist= model.FloatContinuous(MAX_DISTANCE_FOCUS_POINTS, range=(50e-6, 1000e-6))
```

Comment:
missing space before =
```suggestion
        self.focus_points_dist = model.FloatContinuous(MAX_DISTANCE_FOCUS_POINTS, range=(50e-6, 1000e-6))
```